roop/core.py

```diff
@@ -237,7 +244,7 @@ def start() -> None:
 
 
 def destroy() -> None:
-    if roop.globals.target_path:
+    if roop.globals.target_path and state.is_done(roop.globals.target_path) and state.unprocessed_frames_count(roop.globals.target_path) != detect_fc(roop.globals.target_path):
```

Comment:
That needs to be a single function as awell like `state.is_unfinished()`

Reply:
done